root/io/io/TFileBenchmarks.cxx

```diff
@@ -0,0 +1,33 @@
+#include "TROOT.h"   // ROOT::EnableImplicitMT
+
+#include "TFile.h"	
+#include "TFrame.h"
+#include "TRandom3.h"
+#include "TBenchmark.h"
+
+#include "ROOT/RDataFrame.hxx"
+#include "ROOT/RSnapshotOptions.hxx"
+
+#include "benchmark/benchmark.h"
+#include "rootbench/RBConfig.h"
+
+
+static void BM_TFile_Write(benchmark::State &state) {
+	ROOT::EnableImplicitMT();
+	ROOT::RDF::RSnapshotOptions options;
+	options.fCompressionAlgorithm = ROOT::ECompressionAlgorithm::kZLIB;
```

Comment:
Add please tests also for other compression algorithms: LZMA, LZ4.

Reply:
Added